### PR TITLE
Change the implementation of LoadRules in flow rule manager.go

### PR DESCRIPTION
**Pull request by wenxuwan**
### Describe what this PR does / why we need it

The implementation of LoadRules() in rule_manager.go might be not elegant, and the channel ruleChan seems to have no obvious meaning.

### Does this pull request fix one issue?
#71 

### Describe how you did it

Change the function implementation same with flow rule. And remove the unused code in file.

### Describe how to verify it
unit test is ok

### Special notes for reviews

### CODE REVIEW

File: core/system/rule_manager.go [hunk 1]
```diff
@@ -55,12 +34,22 @@ func GetRules() []*SystemRule {
 }
 
 // LoadRules loads given system rules to the rule manager, while all previous rules will be replaced.
-func LoadRules(rules []*SystemRule) (bool, error) {
```
Comment (louyuting) [inline core/system/rule_manager.go]: The code related to system rule manager is duplicated  with https://github.com/alibaba/sentinel-golang/pull/72.
I will review flow rule related code.
It's better to remove system rule related code.
Thanks

File: core/flow/rule_manager.go [hunk 2]
```diff
@@ -25,15 +24,10 @@ var (
 	tcMap        = make(TrafficControllerMap, 0)
 	tcMux        = new(sync.RWMutex)
 
-	ruleChan     = make(chan []*FlowRule, 10)
 	propertyInit sync.Once
```
Comment (louyuting) [inline core/flow/rule_manager.go]: You should delete sync.Once simultaneously.

File: core/flow/rule_manager.go [hunk 3]
```diff
@@ -89,10 +67,10 @@ func onRuleUpdate(rules []*FlowRule) (err error) {
 }
 
 // LoadRules loads the given flow rules to the rule manager, while all previous rules will be replaced.
-func LoadRules(rules []*FlowRule) (bool, error) {
+func LoadRules(rules []*FlowRule) error {
```
Comment (louyuting) [inline core/flow/rule_manager.go]: The pair(bool, error) of return value is required.
bool: indicate whether this Load update the rule map;
error: indicate whether occur error in this load operation.

File: core/system/rule_manager.go (hunk 1 above)
Comment (wenxuwan) [reply core/system/rule_manager.go]: done

File: core/flow/rule_manager.go (hunk 3 above)
Comment (louyuting) [reply core/flow/rule_manager.go]: We might add some cache logic to check whether current value is consistent with last update value. It's under discussion.

Comment (wenxuwan) [reply core/flow/rule_manager.go]: But in existing code logic, it is meaningless. We return it directly with True.

Comment (wenxuwan) [reply core/flow/rule_manager.go]: Ok, i recover the code for future design.

Comment (louyuting) [reply core/flow/rule_manager.go]: Currently, we return it directly with True.
As comments upper, it's sure that we will add a cache logic, but the exact design hasn't been decided yet. 
In order to keep all logic consistent in xxx rule manager, it's better to use a pair(bool, error) return value.

Comment (louyuting) [reply core/flow/rule_manager.go]: Thanks~

**Pull request merged**